localisation/00_zuwc_united_localisation.csv

```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
+EVTDESC_WCRAC_500;You order the most experienced carver you have to sculpt your new body. Together with him, you head to career near the capital. Masons who work there assure that you will not find better stone in the whole country. They tell that there are two types of rock: soft and hard. Your carver responds: it's easier to sculpt soft stone, but a body made of hard one will last you longer and better.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_500;Soft, I don't want to risk.;;;;;;;;;;;;;x
+EVTOPTB_WCRAC_500;A hard will in a hard body.;;;;;;;;;;;;;x
+spiritbinding_deformed_body_tooltip;Your new body is deformed.;;;;;;;;;;;;;x
+spiritbinding_excellent_body_tooltip;Your new body is of excellent quality.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_505_normal;Call the spiritbinders.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_505_fail;Fine, you may live! Call the spiritbinders.;;;;;;;;;;;;;x
+EVTOPTB_WCRAC_505_normal;My soul isn't ready yet.;;;;;;;;;;;;;x
+EVTOPTB_WCRAC_505_fail;Execute him. This body doesn't suit me.;;;;;;;;;;;;;x
+EVTDESC_WCRAC_505;Finally, the carver presents your new body. You slowly walk around the sculpture. [Root.GetBodyQualityResponse];;;;;;;;;;;;;x
+GetBodyQualityResponse_deformed;Even a blind person would notice that the body is damaged. You look at the carver, giving you a smile in return while the guard comes up behind him.;;;;;;;;;;;;;x
+GetBodyQualityResponse_excellent;It looks fabulous. These curves, these shapes resemble the statues of ancient masters. It's a perfect vessel for you indeed. You look at the carver, giving you a smile in return.;;;;;;;;;;;;;x
+GetBodyQualityResponse_normal;It looks fine. You remark sharp angles and slight antisymmetry, but it's barely noticeable. It's a good vessel for you indeed. You look at the carver, giving you a smile in return.;;;;;;;;;;;;;x
+spiritbinding_soul_transferred_tooltip;Your soul is successfully transfered.;;;;;;;;;;;;;x
+spiritbinding_soul_trapped_tooltip;Your soul is trapped.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_510;What it's?!;;;;;;;;;;;;;x
+EVTDESC_WCRAC_510;Spiritbinders gather around you muttering their mantra. You feel like you're falling. Trying to find the ground under your feet, you look down and see your body. [Root.GetSpiritbindingOutcome];;;;;;;;;;;;;x
+GetSpiritbindingOutcome_transferred;The soul float toward your new vessel. Finally, flash. With your new eyes, you watch the old empty body falling to its knees.;;;;;;;;;;;;;x
+GetSpiritbindingOutcome_trapped;You float farther and farther away straight to the firmament. At the last moment of your life, you witness the giant black whirlwind among the clouds.;;;;;;;;;;;;;x
+build_new_vessel;Build New Vessel;;;;;;;;;;;;;x
+build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus you're able to transfer your soul to a new vessel if the current one doesn't suit you anymore. You will be rejuvenated and lose all the body-related traits.;;;;;;;;;;;;;x
```

Comment:
```suggestion
build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus, you're able to transfer your soul to a new vessel if the current one doesn't suit you anymore. You will be rejuvenated and lose all the body-related traits.;;;;;;;;;;;;;x
```

Reply:
Done in 1ab80ba4441f459c577073e118cd7a99e309a891.